apiary.apib

```diff
@@ -762,6 +762,39 @@ Get a list of all work types, sorted alphabetically (on their name).
                 + id: `811a5825-96f4-4318-83c3-2840935c6003` (string)
                 + name: `Planning` (string)
 
+## Templates [/templates]
+
```

Comment:
PP: denote that this is about document layout templates, especially given that this is not within some more specific section in the docs.

Reply:
Any idea how this can be described as clear as possible?